release/charts/charts.go

```diff
@@ -0,0 +1,58 @@
+package charts
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+
+	ecmConfig "github.com/rancher/ecm-distro-tools/cmd/release/config"
+)
+
+// ListLifecycleStatus prints the lifecycle status of the charts
+func ListLifecycleStatus(ctx context.Context, c *ecmConfig.ChartsRelease, branch, chart string) error {
+
+	var branchArg, chartArg string = "", ""
+
+	branchArg = "--branch-version=" + branch
+	if chart != "" {
+		chartArg = "--chart=" + chart
+	}
+
+	err := executeChartsBuildScripts(c.Workspace, "lifecycle-status", branchArg, chartArg)
+	if err != nil {
+		return err
+	}
+
+	fmt.Printf("generated log files for inspection at: \n%s\n", c.Workspace+"/logs/")
+	return nil
+}
+
+func executeChartsBuildScripts(chartsRepoPath string, args ...string) error {
+	// save current working dir
+	ecmWorkDir, _ := os.Getwd()
+
+	// change working dir to the charts repo
```

Comment:
What happens if there's an error?

Reply:
@briandowns 
There is no way to continue the execution if there is an error when discovering the current dir and changing dirs.